followthemoney/cli/ocds.py

```diff
@@ -137,8 +140,19 @@ def convert_release(release):
 def convert_record(record):
     published_date = clean_date(record.pop('publishedDate', None))
     publisher = record.pop('publisher', {}).get('name')
-    for release in record.get('releases', []):
-        for entity in convert_release(release):
+    if record.get('tag'):
+        for entity in convert_release(record):
```

Comment:
Would it make sense to rename `convert_release` to a more generic term that covers both releases and records? `convert_item`?

Reply:
Yes.